tardis/transport/montecarlo/tests/test_trackers.py

```diff
@@ -0,0 +1,144 @@
+from copy import deepcopy
```

Comment:
Is this import used anywhere?

Reply:
Removed this import as it was not used elsewhere.